manifests/config.pp

```diff
@@ -61,11 +61,6 @@
 
     'none'     : {
       # Don't configure apache, gunicorn or nginx. Leave all webserver configuration to something external.
-      if !$::graphite::gr_web_user or !$::graphite::gr_web_group {
-        fail('having $gr_web_server => \'wsgionly\' requires use of $gr_web_user and $gr_web_group')
```

Comment:
I admit that this message is misleading. We should change it to something like:
`fail('Having $gr_web_server => \'none\' requires use of $gr_web_user and $gr_web_group to set correct file owner for your own webserver setup.')`